test/lib/render-helpers.js

```diff
@@ -5,6 +5,7 @@ import { mount } from 'enzyme'
 import { MemoryRouter } from 'react-router-dom'
 import PropTypes from 'prop-types'
 import { LegacyI18nProvider } from '../../ui/app/contexts/i18n'
+import { LegacyMetaMetricsProvider } from '../../ui/app/contexts/metametrics'
```

Comment:
Would it be more appropriate to setup a Test MetaMetrics Instance of this that doesn't send events?

Reply:
The `noopSegment` instance should typically be used in a test/dev environment, so this already doesn't send events.

This was implemented after you wrote this comment initially I believe.